### DISCUSSION TITLE
How create multiple html  pages from a  static database file like bibtex,  json or csv?

### DISCUSSION THREAD

**Discussion by matteopedani** [quarto-dev/quarto-cli#8693] [Q&A]
### Description

I have a database of book I  made using  Zotero. Zotero can export in varius format, like  csv, bibtex, json  and more....
I will create a book web page for each book record. 
If I have extra field, (not from ISBN), there is one best way, to add data to records,  to show on book web pages?

**Comment by mcanouil** [comment]
There are many ways, one would be to create a YAML file containing your elements and using the listings features, see:
- https://quarto.org/docs/websites/website-listings.html
- https://quarto.org/docs/websites/website-listings-custom.html

Note that I don't understand what a "book web page" is as a book is by definition a multi-page thing.
